README.md

```diff
@@ -696,60 +631,60 @@ Chocolatey v0.9.9.0+.
 #### Properties/Parameters
 
 ##### `name`
-(**Namevar**: If ommitted, this attribute's value will default to the resource's
-title.)
 
-The name of the source. Used for uniqueness. Will set the location to this value if location is unset.
+Specifies the name of the source. Used for uniqueness. Also sets the `location` to this value if `location` is unset. Valid options: String. Default: The resource's title.
```

Comment:
Also lost that this is required. Perhaps the note of the default value is sufficient.